src/core/SablierLockup.sol

```diff
@@ -236,15 +252,16 @@ contract SablierLockup is ISablierLockup, SablierLockupBase {
     /// @inheritdoc ISablierLockup
     function createWithTimestampsLL(
         Lockup.CreateWithTimestamps calldata params,
-        uint40 cliffTime
+        uint40 cliffTime,
+        LockupLinear.UnlockAmounts calldata unlockAmounts
```

Comment:
Lets change the order of input parameters similar to other functions:
1. Lockup.CreateWithTimestamps calldata params
2. LockupLinear.UnlockAmounts calldata unlockAmounts
3. uint40 cliffTime

Reply:
my rational was that the cliff time would be more common than the unlock amounts, but i don't have a strong opinion

what is your reasoning for this?

Reply:
> my rational was that the cliff time would be more common than the unlock amounts

Mb but guessing at this stage is hard, unlock at start is also a very popular distribution strategy. It can also be popular for instant airdrops (no more 1 sec vesting hack). I tend to follow the below rule:

1. Have structs first followed by simple value types.
2. Order structs alphabetically using variable names.
3. Order value types alphabetically using variable names.

If you have other ideas, I'd love to hear that. My reasoning is weak as well but inspired by the fact that its sometimes arbitrary how we order input parameters in the function names. So having a rule would help. We can also place it in [Sablier Coding Practices](https://sablier.notion.site/Sablier-Coding-Practices-Contracts-2f5920cf497646b4a2ea982c1bacb5a8).

Reply:
ended up making this change